### PR TITLE
Refactored Dockerfile

### PR DESCRIPTION
**Pull request by Lerentis**
Hey there,

i saw that your Dockerfile depended on Debian. I refactored it to depend on Alpine which is much smaller. This should not change anything functional, it just decreases the image size

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,7 +18,7 @@ $(MAKEFILE):
 
 # we still need to do this for windows
 bblfsh-client:
-	cd vendor/gopkg.in/bblfsh/client-go.v2 && make dependencies
+	cd vendor/gopkg.in/bblfsh/client-go.v3 && make dependencies
```
Comment (jfontan) [inline Makefile]: Why the change in client-go version? If I am not mistaken we are still using v2.

Comment (erizocosmico) [reply Makefile]: Nope, we're using v3 in master

File: Dockerfile [hunk 2]
```diff
@@ -8,15 +8,13 @@ ENV GITBASE_PASSWORD=""
...
+ENV TINI_VERSION v0.18.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-static-amd64 /tini
 RUN chmod +x /tini
 
-RUN apt-get update \
-  && apt-get -y --no-install-recommends install libxml2 git \
-  && apt-get clean \
-  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
-
+RUN echo 'http://dl-cdn.alpinelinux.org/alpine/v3.1/main' >> /etc/apk/repositories && \
```
Comment (erizocosmico) [inline Dockerfile]: shouldn't this be `v3.8` to match the alpine version in use?

**Comment by Lerentis** [comment]
I was not sure about the version of libxml so I forced it with this repository. If the version of libxml does not matter I can remove the whole repository there

-----Original Message-----

**Comment by erizocosmico** [comment]
I would say if it compiles and works correctly, there's no libxml version requirement afaik.

**Comment by Lerentis** [comment]
I removed the version pin and with that the extra repository is not needed anymore
